melodic/distribution.yaml

```diff
@@ -814,6 +814,28 @@ repositories:
       url: https://github.com/mavlink/mavlink-gbp-release.git
       version: release/melodic/mavlink
     status: maintained
+  mavros:
+    doc:
+      type: git
+      url: https://github.com/mavlink/mavros.git
+      version: master
+    release:
+      packages:
+      - libmavconn
+      - mavros
+      - mavros_extras
+      - mavros_msgs
+      - test_mavros
```

Comment:
just checking: are you releasing this test package on purpose ? if you don't wish to release it, you can place a `melodic.ignored` file containing the package name in your release repository.

Reply:
Good point, sorry I missed that in the review.

Reply:
That package provide some hand-tests that may be handy for users.

Reply:
sounds good :+1: , thanks for confirming